build/yaml/pipelines/signed.yaml

```diff
@@ -20,11 +20,13 @@ stages:
       IndexSources: true
       SignType: $(SignType)
       Configuration: Release
+      UseInternalTasks: true
```

Comment:
>UseInternalTasks [](start = 6, length = 16)

Nit: consider something like "IsOfficialBuild" (though I think we have a variable for that already), or "IsMicrobuildInternal".